spec/spec_helper.rb

```diff
@@ -14,24 +14,41 @@
   end
 end
 
+TEST_CREDENTIALS_DEFAULTS = {
+  :vmware_cloud_defaults => {:host => "vmwarecloudhost", :userid => "VMWARE_CLOUD_USERID", :password => "VMWARE_CLOUD_PASSWORD"},
+  :vmware_infra_defaults => {:hostname => "HOSTNAME"},
+  :vmware_tanzu_defaults => {:hostname => "vmware-tanzu-hostname", :userid => "VMWARE_TANZU_USERID", :password => "VMWARE_TANZU_PASSWORD"}
```

Comment:
I'm open to make the constant have the `DEFAULTS` name an simplify the keys to no longer have defaults in the name to avoid redundancy.  Let me know your thoughts.